### ISSUE TITLE
Sharp doesn't work with vips 8.0.0+

### ISSUE THREAD

**Issue by jgillman** [jimmynicol/image-resizer#51]
I suspect this is something that Sharp would need to fix, but I'm not entirely clear if this is a problem with Sharp or a problem with image-resizer's integration with Sharp.

I updated vips with brew to the latest version (8.0.2) and afterwards got this error message when starting up the server:

```
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: dlopen(/Users/jgillman/Code/myproject/node_modules/image-resizer/node_modules/sharp/build/Release/sharp.node, 1): Library not loaded: /usr/local/lib/$
ibvips.40.dylib
  Referenced from: /Users/jgillman/Code/myproject/node_modules/image-resizer/node_modules/sharp/build/Release/sharp.node
  Reason: image not found
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/jgillman/Code/myproject/node_modules/image-resizer/node_modules/sharp/index.js:11:13)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/jgillman/Code/myproject/node_modules/image-resizer/src/streams/filters/blur.js:3:13)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
```

# Downgrade vips with brew

For anyone with the same problem, you can install the 7.42.3 version of vips with brew like so:

```sh
brew update
brew remove vips
cd /usr/local/Library
git checkout 6fc408b25a Taps/homebrew/homebrew-science/vips.rb
brew install homebrew/science/vips --with-webp --with-graphicsmagick
```

If you later need to install the most recent version you can reset your change:

```sh
brew remove vips
cd /usr/local/Library
git reset HEAD Taps/homebrew/homebrew-science/vips.rb
brew install homebrew/science/vips --with-webp --with-graphicsmagick
```

For reference, you can always check your vips version with `vips --version`.

The version you are installing from the code above this commit: https://github.com/Homebrew/homebrew-science/commit/6fc408b25a0c7f0afc4a978b5a1165b28175c629

**Comment by jimmynicol** [comment]
@jgillman 

Sorry for taking a while to respond to this, but I dont believe this is an issue at all. I just ran the library against `v8.0.2` with no issues. I suspect that you did you re-install sharp when you upgraded the library. Sharp needs to re-build is c bindings against the new vips and it will fail otherwise. 

I just bumped the repo to `v1.0.2` to include a newer sharp. If you upgrade I would recommend blowing away your `/node_modules` and re-installing everything.
